contracts/utils/Pausable.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { IPausable } from '../interfaces/IPausable.sol';
+
+/**
+ * @title Pausable
+ * @notice This contract provides a mechanism to halt the execution of specific functions
+ * if a pause condition is activated.
+ */
+contract Pausable is IPausable {
```

Comment:
Same here
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Pausable.sol

but can keep only the methods we care about